### DISCUSSION TITLE
Welcome to ESM discussions!

### DISCUSSION THREAD

**Discussion by tomsercu** [facebookresearch/esm#31] [General]
The Discussions forum is a great spot to ask questions about ESM, like "how do I do X?". The advantage over github issues for this, is that those conversations are more easily findable by others with the same or related questions.

We also welcome any general feedback about ESM in this forum, or discussion of new applications of ESM that are useful for others.

Please adhere to the [Code of Conduct](https://engineering.fb.com/codeofconduct/) and the GitHub Community Guidelines when discussing in this forum.

Thanks for your interest in ESM!

**Comment by HHLA917A** [comment]
The structure Meta returns for the protein MPDZ (for example) is small and trivial relative to the partially solved structure and the predicted structure by AlphaFold. As such, there is far inferior to those resources.
